fhir-persistence-schema/deployPriorVersions.sh

```diff
@@ -9,8 +9,9 @@ set -e
 
 # Version 4.0.1 of the fhir-persistence-schema cli doesn't support derby, but starting with 4.1.0 we should do something like this:
```

Comment:
let's document this file in Db2SchemaMigrations.md

Reply:
added it to a new `Testing migrations` section

Reply:
awesome!